src/pages/token-metadata/update.md

```diff
@@ -6,17 +6,85 @@ description: Learn how to update Assets on Token Metadata
 
 The update authority of an asset can update its **Metadata** account using the **Update** instruction as long as the **Is Mutable** attribute is set to `true`. The **Update** instruction requires the **Update Authority** to sign the transaction and can update the following attributes of the **Metadata** account:
 
-- **Data**: The object that defines the Name, Symbol, URI, Seller Fee Basis Points and the array of Creators of the asset. Note that the update authority can only add and/or remove unverified creators from the Creators array. The only exception is if the creator is the update authority, in which case the added or removed creators can be verified.
-- **Primary Sale Happened**: A boolean that indicates whether the asset has been sold before.
-- **Is Mutable**: A boolean that indicates whether the asset can be updated again. When changing this to `false`, any future updates will fail.
-- **Collection**: This attribute enables us to set or clear the collection of the asset. Note that when setting a new collection, the `verified` boolean must be set to `false` and [verified using another instruction](/token-metadata/collections).
-- **Collection Details**: This attribute enables us to set or clear the collection details of the asset.
-- **Rule Set**: This attribute enables us to set or clear the rule set of the asset. This is only relevant for [Programmable Non-Fungibles](/token-metadata/pnfts).
+## Updatable Fields
 
 Note that certain delegated authorities can also update the **Metadata** account of assets as discussed in the "[Delegated Authorities](/token-metadata/delegates)" page.
 
```

Comment:
I really like the way you broke each attribute/parameter out and explained it in a mini-section.  Easier to read in the rendered version.

One slight thing I could think of would be to add some kind of text in the "##Updatable Fields" section indicating that these are the individual attributes that can be provided to the updateV1 method.

There is already a statement lower between the two code examples, and it does become obvious once you see the code examples below, but if you are looking at this the first time, you just see the data object, primarySaleHappened, isMutable, etc. and you don't get the "establishing shot" until you see the updateV1 code example.  Might be slightly confusing for a newcomer.  

I think just saying something like "Here's the different attributes that can be sent to the updateV1 method" or something like that would work.